polling/testservice.go

```diff
@@ -0,0 +1,25 @@
+package polling
```

Comment:
Can you add a README file in this top-level `polling` dir that links to the three child samples? Also, please update root README to point to this sample dir.